test/jdk/com/sun/jdi/InvokeHangTest.java

```diff
@@ -214,7 +220,7 @@ protected void runTests() throws Exception {
         targetClass = bpe.location().declaringType();
         mainThread = bpe.thread();
         EventRequestManager erm = vm().eventRequestManager();
-        final Thread mainThread = Thread.currentThread();
+        final Thread mainTestThread = Thread.currentThread();
```

Comment:
This local variable was conflicting with an instance field of the same name. See line 135. I ran into issues when I added some debugging code to this method that referenced the other `mainThread`, so I did a rename.